var/spack/repos/builtin/packages/py-torchvision/package.py

```diff
@@ -42,3 +42,5 @@ class PyTorchvision(PythonPackage):
     # Many of the datasets require additional dependencies to use.
     # These can be installed after the fact.
     depends_on('py-scipy', type='test')
+
+    depends_on('ffmpeg@3.1:')
```

Comment:
```suggestion
    depends_on('ffmpeg@3.1:', when='@0.4.2:')
```
https://github.com/pytorch/vision/releases/tag/v0.4.2